linux_os/guide/system/accounts/accounts-physical/disable_ctrlaltdel_reboot/rule.yml

```diff
@@ -41,6 +42,7 @@ references:
     srg: SRG-OS-000324-GPOS-00125,SRG-OS-000480-GPOS-00227
     stigid@rhel7: RHEL-07-020230
     stigid@sle12: SLES-12-010610
+    stigid@sle15: SLES-15-040060
     stigid@sle12: SLES-12-010611
```

Comment:
I think only one of these @sle12 identifiers will be considered

Reply:
Yes, the original in the original disa SLES-12-010611 was a replete of SLES-12-010610, but SLES-12-010611 seems to be gone now. Will remove.